client/my-sites/plans/plan-overview/plan-feature/style.scss

```diff
@@ -0,0 +1,17 @@
+.plan-features__feature {
+	@include plan-overview-row();
+}
+
+.plan-features__feature-button {
```

Comment:
We should update the names here to match the component name, e.g. `.plan-feature__button`